db/migrate/20150729170936_add_items.rb

```diff
@@ -0,0 +1,12 @@
+class AddItems < ActiveRecord::Migration
+  def change
+    create_table :items do |t|
+      t.string :title, null: false
+      t.string :description, null: false
+      t.string :artist, null: false
```

Comment:
On second thought we might want to make another model for this. Since it is possible we want to encompass some of the behavior and allow artists to submit. Aka a user who is an artist